frontend/ui/data/onetime_migration.lua

```diff
@@ -400,5 +400,10 @@ if last_migration_date < 20220523 then
     migrateSettingsName("highlight_long_hold_threshold", "highlight_long_hold_threshold_s")
 end
 
+-- #9104
+if last_migration_date < 20220607 then
+    os.remove("afterupdate.marker")
+end
```

Comment:
Do we care ? Can't we just let it lie there - like we do when any previous installation of KOReader had a file that we removed or renamed.

Reply:
No problem for me, but iirc @pazos wanted this.

Reply:
@pazos : still wanting this cleanup?

Reply:
Hi, I didn't requested that. (I see my comment is not available anymore, maybe because force push?)

I've said: please don't rename the file as the old one is going to be left on user devices. Changing the contents of the file is ok.

I still think it makes more sense to keep the old file. Its intent was: an after update marker. The new name "update_once" doesn't match the intent of the file but the usage we're going to do with the file.

So, my order of preference is:

1. Keep the old file, change contents. Avoid migration.
2. Keep the old file and the new file. Avoid migration.

I would vote for 1 unless there's a very specific reason for the name change. **But** I already requested my quote of changes for this PR so I'm going to be fine no matter what's happen.

Reply:
Given that I didn't say much yet and this line stands out like a sore thumb, let's put it on my quota then. ;-)

Reply:
@Frenzie: To delete or keep?

Reply:
delete, I think
> this line stands out like a sore thumb

Reply:
Although I'm smart and I enjoy hidden clues, I would sometimes be grateful for clear statements. 

Especially after a 100+ conversation.

Reply:
@poire-z  Thanks.

Reply:
Apologies, I thought I was crystal clear:

> Keep the old file, change contents. Avoid migration.

Reply:
PS That's a generic statement as well.